File: guides/voice/record-calls-guide/record-outgoing-call/example-1.7.x.java
// Install the Java helper library from twilio.com/docs/java/install
import com.twilio.Twilio;
import com.twilio.rest.api.v2010.account.Call;
import com.twilio.type.PhoneNumber;

import java.net.URI;
import java.net.URISyntaxException;

public class Example {

    // Find your Account Sid and Token at twilio.com/user/account
    public static final String ACCOUNT_SID = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
    public static final String AUTH_TOKEN = "your_auth_token";

    public static void main(String[] args) throws URISyntaxException {
        Twilio.init(ACCOUNT_SID, AUTH_TOKEN);

        Call call = Call.creator(
                new PhoneNumber("+14155551212"),
                new PhoneNumber("+15017250604"),
                new URI("http://demo.twilio.com/docs/voice.xml")
                )
                .setRecord(true)
                .create();

        System.out.println(call.getSid());
    }
}

File: guides/voice/record-calls-guide/record-outgoing-call/example-1.7.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
// To set up environmental variables, see http://twil.io/secure
$accountSid = getenv('TWILIO_ACCOUNT_SID');
$authToken = "your_auth_authToken";
$client = new Client($accountSid, $authToken);

$call = $client->calls->create(
    "+14155551212", "+12349013030",
    array("url" => "http://demo.twilio.com/docs/voice.xml", "record" => true)
);

echo $call->sid;
